api/v1alpha1/openapi.yaml

```diff
@@ -2502,6 +2532,231 @@ components:
           type: string
           description: 'Status of the operation. One of: "Success" or "Failure". More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status'
       description: Status is a return value for calls that don't return other objects.
+    DeviceApplicationsStatus:
+      type: object
+      properties:
+        status:
+          $ref: '#/components/schemas/DeviceApplicationSummary'
+          description: 'Summary status of device applications.'
+        applications:
+          type: array
+          description: 'Status of device applications.'
+          items:
+            $ref: '#/components/schemas/DeviceApplication'
```

Comment:
I think we should not focus on the "Application" statuses yet. This is something we may want to look at when we have the next MVP version.

Because we need to define what an application is, and how do we monitor those.

While for now we were just monitoring simple systemd and container processes.

All the other status bits

Reply:
@fzdarsky ?

Reply:
Ok, as per slack conversation, we want to have a definition of the API now.